src/server/server.ts

```diff
@@ -385,9 +388,9 @@ registerRoute('post', '/export', (req, res) => {
     // to avoid unhandled exception-pausing on debugging.
     return Promise.all([
         generateSnippetHtmlData(
-            { scriptToCompile: snippet.script, ...extractCommonCompileData(snippet) },
+            {scriptToCompile: snippet.script, ...extractCommonCompileData(snippet) },
             'script',
-            true /*isExternalExport*/, strings),
+            true /*isExternalExport*/, strings, true /*isInsideOffice*/),
```

Comment:
multiple boolean parameters that need explanation might be a code smell for making the function accept an options bag instead.